russian/wca-regulations.md

```diff
@@ -509,11 +510,40 @@
+    - Y2a) Посетитель - любой человек, посещающий соревнование: участник, официальное лицо или гость.
+    - Y2b) Фиксированная рассадка участников - способ проведения соревнования, при котором участники делают свои попытки в данном раунде на одном и том же месте для сборки.
+- Y3) Помимо требований, перечисленных в [Положении 2a](regulations:regulation:2a), к участнику дополнительно могут предъявляться следующие требования, если Делегат WCA считает их необходимыми, а также о них было объявлено заранее:
+    - Y3a) Носить защитную медицинскую маску в месте проведения соревнования.
+    - Y3b) Не иметь контакта с человеком с COVID-19 по крайней мере в течение 14 дней до начала соревнования.
+    - Y3c) Не иметь COVID-19 или испытывать симптомы COVID-19 (определенные [Всмемирной организацией здравоохранения](https://www.who.int/)) по крайней мере в течение 14 дней до начала соревнования.
+    - Y3d) Соблюдать остальные разумные меры безопасности, введённые Всемирной организацией здравоохранения или местными властями.
+- Y4) Всем участникам и официальным лицам следует приносить свои собственные ручки на соревнование.
+- Y5) Судьи могут принести свой собственный секундомер или телефон для измерения времени преинспекции, которые следует проверить Делегату WCA.
+- Y6) Окружающая обстановка:
+    - Y6a) Если Делегат WCA считает это необходимым, в месте проведения соревнования не должно быть зоны для участников.
+    - Y6b) Участникам не следует трогать поверхность стола для сборки, если только они не в процессе выполнения официальной попытки.
+- Y7) Ниже описаны дополнительные процедуры, замещающие соответствующие положения в [Статье A](regulations:article:A), [Статье B](regulations:article:B) и [Статье H](regulations:article:H). Эти процедуры не применимы к [Статье E](regulations:article:E), Сборке на количество ходов.
+    - Y7a) На соревнованиях следует использовать фиксированную рассадку участников (см. [Положение Y2b](regulations:regulation:Y2b)).
+    - Y7b) Судье следует избегать касания головоломки в любой ситуации. Исключение: в случае сборки Clock, судье следует поднять головоломку, чтобы убедиться, что обе стороны головоломки собраны.
+    - Y7c) Перемешка головоломок:
+        - Y7c1) Когда участника вызывают для участия в раунде, после того как он продезинфицировал свою головоломку, участник сдает её скрамблеру, в собранном состоянии. Участник затем ждёт своих попыток за своим местом.
+    - Y7d) Преинспекция:
+        - Y7d1) Судья просит участника приготовить таймер, включив его и сбросив при необходимости. Участник затем включает и сбрасывает таймер, если необходимо. Судье не следует касаться таймера.
+    - Y7e) Запись результатов:
+        - Y7e1) Участник должен проверить записанный результат и подписать его (или поставить инициалы) на карточке собственной ручкой, принимая, что результат верный, полный, правильно отформатирован и хорошо читаемый.
+        - Y7e2) Участнику не следует касаться карточки для результатов.
```

Comment:
```suggestion
        - Y7e2) Если участник не согласен с результатом или судья отказывается подписать результат, Делегат WCA должен решить их спор.
```

Reply:
Again, this isn't consistent with the current commits in [this PR](https://github.com/thewca/wca-regulations/pull/980/)

Reply:
- Y7e2) If the competitor or the judge refuses to accept and verbally approve or sign, respectively, the score sheet, the WCA Delegate must resolve the dispute. 
        - Y7e3) The competitor should not tell their judge the result is correct before the judge has recorded the attempt. The competitor is ultimately responsible for ensuring that the judge has signed an attempt after they have verbally approved it. If an attempt is found to be missing the judge’s signature after the judge delivered the score sheet to the score taker (see Regulation A7f), it will be considered unfinished (DNF).